kolibri/core/assets/src/state/getters.js

```diff
@@ -1,29 +1,50 @@
 import { UserKinds, MaxPointsPerContent } from '../constants';
-import cookiejs from 'js-cookie';
+import some from 'lodash/some';
 
-function isUserLoggedIn(state) {
-  return state.core.session.kind[0] !== UserKinds.ANONYMOUS;
+// ROLES
+function isAdmin(state) {
+  return state.core.session.kind.includes(UserKinds.ADMIN);
 }
 
-function isSuperuser(state) {
-  return state.core.session.kind[0] === UserKinds.SUPERUSER;
+function isCoach(state) {
+  return state.core.session.kind.includes(UserKinds.COACH);
 }
 
-function isAdmin(state) {
-  return state.core.session.kind[0] === UserKinds.ADMIN;
+function isLearner(state) {
+  return state.core.session.kind.includes(UserKinds.LEARNER);
 }
 
-function isCoach(state) {
-  return state.core.session.kind[0] === UserKinds.COACH;
+function isUserLoggedIn(state) {
+  return !state.core.session.kind.includes(UserKinds.ANONYMOUS);
 }
 
-function isLearner(state) {
-  return state.core.session.kind[0] === UserKinds.LEARNER;
+function getUserRole(state) {
+  if (isAdmin(state)) {
+    return UserKinds.ADMIN;
+  } else if (isCoach(state)) {
+    return UserKinds.COACH;
+  } else if (isLearner(state)) {
+    return UserKinds.LEARNER;
+  }
+  return UserKinds.ANONYMOUS;
 }
 
+// PERMISSIONS
 function canManageContent(state) {
   return state.core.session.can_manage_content;
 }
+function isSuperuser(state) {
+  return state.core.session.kind.includes(UserKinds.SUPERUSER);
+}
+function getUserPermissions(state) {
+  const permissions = {};
+  permissions.can_manage_content = state.core.session.can_manage_content;
+  return permissions;
+}
+
+function userHasPermissions(state) {
+  return some(getUserPermissions(state), permission => permission === true);
```

Comment:
`some(getUserPermissions(state), /* no predicate */)` should be enough. If the key `permissions` is Boolean, then the predicate for `some` is basically the identity, which is the default.